python/src/fragment.rs

```diff
@@ -50,6 +52,29 @@ impl FileFragment {
         format!("LanceFileFragment(id={})", self.fragment.id())
     }
 
+    #[staticmethod]
+    #[pyo3(signature = (filename, schema, fragment_id))]
+    fn create_from_file(
+        filename: &str,
+        schema: &PyAny,
+        fragment_id: usize,
+    ) -> PyResult<FragmentMetadata> {
+        let rt = Runtime::new()?;
+        let arrow_schema = ArrowSchema::from_pyarrow(schema)?;
```

Comment:
We can make the PyArrow schema be part of the signature.
```suggestion
        schema: PyArrowType<ArrowSchema>,
        fragment_id: usize,
    ) -> PyResult<FragmentMetadata> {
        let rt = Runtime::new()?;
        let arrow_schema = schema.0;
```

May need to add:

```rust
use deltalake::arrow::pyarrow::PyArrowType;
```